ci-operator/step-registry/hypershift/aws/setup-nested-management-cluster/hypershift-aws-setup-nested-management-cluster-chain.yaml

```diff
@@ -3,6 +3,9 @@ chain:
   steps:
   - as: create-management-cluster
     cli: latest
+    leases:
```

Comment:
Docs for [explicit lease configuration][1], for folks like me who aren't all that familiar with this syntax.

[1]: https://docs.ci.openshift.org/docs/architecture/step-registry/#explicit-lease-configuration